blog/2019-02-07-KIEServerOptaPlannerTaskAssignment.adoc

```diff
@@ -0,0 +1,215 @@
+= KIE Server OptaPlanner Task Assignment
+:page-interpolate: true
+:awestruct-author: MusaTalluzi
+:awestruct-layout: blogPostBase
+:awestruct-tags: [useCase, task assigning, kie-server]
+
+https://docs.optaplanner.org/7.17.0.Final/optaplanner-wb-es-docs/html_single/#_ch.kie.server[KIE Server]
+is a standalone server component that can be used to instantiate and execute rules and processes.
+In this blog, you will learn how to build an OptaPlanner service that implements continuous and real-time planning to solve the
+Task Assignment problem and deploy it to the KIE Server.
+
+
+== https://github.com/kiegroup/kie-server-optaplanner-task-assignment/tree/master/task-assignment-kjar[Task Assignment KJAR]
```

Comment:
Maybe use a tag instead of master. This will inevitably result in a broken link once the class is moved somewhere else on master :slightly_smiling_face:

Reply:
Agreed that this link is brittle, but not sure if the tag would be of much help. I 'd just reference the repository source code itself. The user should find their way from their if the modules are well named. Wdyt?